content/lib/pencil-tracer.js

```diff
@@ -83,8 +83,24 @@
       return null;
     };
 
+    CoffeeScriptInstrumenter.prototype.isFunctionDef = function(node) {
+      return node instanceof this.nodeTypes.Assign && !node.context && node.variable instanceof this.nodeTypes.Value && node.variable.base instanceof this.nodeTypes.Literal && node.variable.properties.length === 0 && node.value instanceof this.nodeTypes.Code;
```

Comment:
It would be nice if this were split into multiple lines. Easier to read, and probably via indentation you could help me understand the structure.